packages/dd-trace/src/config.js

```diff
@@ -236,6 +237,25 @@ function reformatSpanSamplingRules (rules) {
 
 class Config {
   constructor (options = {}) {
+    // Read & apply the YAML configuration
+    const libconfig = libdatadog.maybeLoadWASM('library_config')
+    if (libconfig != null) {
```

Comment:
Not sure I understand this condition. My understanding is that the native config would be replacing the config we currently do in process. It doesn't quite make sense to me then to allow it to be missing, it should always be present so that we can remove the JS code.

Reply:
> My understanding is that the native config would be replacing the config we currently do in process.
The current way of configuring the tracing library will not be removed. We will simply add new configuration sources.

This is mostly a failsafe to avoid crashing the app if anything happens.